### PR TITLE
V7 - Split Integration Tests workflow

### PR DESCRIPTION
**Pull request by gururajsh**
Thank you for contributing to the CF CLI! Please read the following:

* Please make sure you have implemented changes in line with the [contributing guidelines](https://github.com/cloudfoundry/cli/blob/main/.github/CONTRIBUTING.md)
* We're not allowed to accept any PRs without a signed CLA, no matter how small.
If your contribution falls under a company CLA but your membership is not public, expect delays while we confirm.
* All new code requires tests to protect against regressions.
* Contributions must be made against the appropriate branch. See the [contributing guidelines](https://github.com/cloudfoundry/cli/blob/main/.github/CONTRIBUTING.md)
* Contributions must conform to our [style guide](https://github.com/cloudfoundry/cli/wiki/CLI-Product-Specific-Style-Guide). Please reach out to us if you have questions.

## Does this PR modify CLI v6, CLI v7, or CLI v8?

Please see the contribution doc above or review [Architecture Guide](https://github.com/cloudfoundry/cli/wiki/Architecture-Guide).

## Description of the Change

We must be able to understand the design of your change from this description.
Keep in mind that the maintainer reviewing this PR may not be familiar with or
have worked with the code here recently, so please walk us through the concepts.

## Why Is This PR Valuable?

What benefits will be realized by the code change? What users would want this change? What user need is this change addressing? 

## Why Should This Be In Core?

Explain why this functionality should be in the cf CLI, as opposed to a plugin. 

## Applicable Issues

List any applicable GitHub Issues here

## How Urgent Is The Change?

Is the change urgent? If so, explain why it is time-sensitive.

## Other Relevant Parties

Who else is affected by the change?

### CODE REVIEW

**Review by a-b — CHANGES_REQUESTED** [review]

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -1,33 +1,17 @@
...
-      - 'README.md'
+  workflow_run:
+    workflows: [Build Binaries]
+    types: [completed]
 
 permissions:
   contents: read
 
-jobs:
+jobs:
```
[the remark this answers is not in the document]
Comment (gururajsh) [reply .github/workflows/integration.yml:11]: Which one is the typo?

File: .github/workflows/integration.yml [hunk 2]
```diff
@@ -55,7 +39,7 @@ jobs:
           min_capi=($(yq .cf-d-capi-version-min build_data.yml))
           echo "Minimum CAPI version: ${min_capi}"
           echo "min-capi=${min_capi}" >> $GITHUB_OUTPUT
-
+
```
[the remark this answers is not in the document]
Comment (gururajsh) [reply .github/workflows/integration.yml:42]: Ok.. Can create a story for this in tracker

**Review by a-b — APPROVED** [review]
LGTM

**Review by a-b — APPROVED** [review]
LGTM

**Pull request merged**